src/user/UsersSection.tsx

```diff
@@ -376,7 +380,7 @@ export default function UsersSection() {
                   />
                 )
               }
-              toolbarItem={toolbar}
+              toolbarItem={isManager ? toolbar : <span />}
```

Comment:
Instead of using a span element here perhaps you can pass in `undefined` explicitly to accomplish the same result without rendering an element?
```suggestion
              toolbarItem={isManager ? toolbar : undefined}
```